_includes/partials/hori-social.html

```diff
@@ -1,16 +1,34 @@
   <div>
     <ul>
-      <a class="white-text" href="mailto:community@layer5.io">
-        <img class="small-on-med-and-down"width="40px" src="/assets/images/email.svg" /></a>
-      <a href="{{site.slack}}">
-        <img class="small-on-med-and-down"width="40px" src="/assets/images/slack.svg" /></a>
-      <a href="{{site.twitter.url}}">
-        <img class="small-on-med-and-down"width="40px" src="/assets/images/twitter.svg" /></a>
-      <a href="{{site.github}}">
-        <img class="small-on-med-and-down"width="40px" src="/assets/images/github.svg" /></a>
-      <a href="{{site.youtube}}">
-        <img class="small-on-med-and-down"width="40px" src="/assets/images/youtube.svg" /></a>
-      <a href="{{site.docker}}">
-        <img class="small-on-med-and-down devicon-docker-plain" width="40px" src="/assets/images/docker.svg" /></a>
+      <div class="tooltip">
+      <a class="white-text " href="mailto:community@layer5.io">
+        <img class="small-on-med-and-down tooltip"width="40px" src="/assets/images/email.svg"  />
+        <span class="tooltiptext">Get connected with the Layer5 community</span></a></div>
+      
+      <div class="tooltip">
+      <a  href="{{site.slack}}">
+        <img class="small-on-med-and-down"width="40px" src="/assets/images/slack.svg" />
+        <span class="tooltiptext">Join the Layer5 Slack</span></a></div>
+        
+        <div class="tooltip">
+     <a  href="{{site.twitter.url}}">
+       <img class="small-on-med-and-down"width="40px" src="/assets/images/twitter.svg" />
+       <span class="tooltiptext">Follow Layer5 on Twitter</span></a></div>
+          
+          <div class="tooltip">
+      <a  href="{{site.github}}">
+        <img class="small-on-med-and-down"width="40px" src="/assets/images/github.svg" />
+       <span class="tooltiptext">Contribute to Layer5 projects</span></a></div >
+      
+      <div class="tooltip">
+      <a  href="{{site.youtube}}">
+        <img class="small-on-med-and-down"width="40px" src="/assets/images/youtube.svg" />
+        <span class="tooltiptext">Watch community meeting recordings</span></a></div>
+      
+        <div class="tooltip">
+      <a  href="{{site.docker}}">
+        <img class="small-on-med-and-down devicon-docker-plain" width="40px" src="/assets/images/docker.svg" />
+        <span class="tooltiptext">Access Docker images for Layer5 projects</span></a></div>
```

Comment:
Docker was accidentally misspelled as "Doker". I corrected this and updated to include a call to action for the user.

Reply:
Sorry for the typo.